src/site/content/en/blog/introducing-model-viewer/index.md

```diff
@@ -1,18 +1,19 @@
 ---
 layout: post
-title: 'Announcing <model-viewer> 1.0'
+title: 'Announcing <model-viewer> 1.1'
 subhead: 3D models for your web page as easily as writing HTML.
 authors:
   - rkochman
   - joemedley
 description: |
-3D models are more relevant than ever. Retailers bring in-store shopping
-experiences to customers' homes. Museums let anyone see artifacts from anywhere.
-3D is difficult to do without a deep knowledge of 3D technologies or third-party
-hosting. <model-viewer> 1.0 makes these tasks as easy as writing HTML.
-date: 2020-08-20
+  3D models are more relevant than ever. Retailers bring in-store shopping
+  experiences to customers' homes. Museums let anyone see artifacts from anywhere.
+  3D is difficult to do without a deep knowledge of 3D technologies or third-party
+  hosting. <model-viewer> 1.1 makes these tasks as easy as writing HTML.
+date: 2020-08-21
+updated: 2020-08-21
 hero: hero.jpg
-alt: A 3d image of a shark.
+alt: A 3D image of an astronaut.
```

Comment:
I thought we decided not to use the astronaut.

Reply:
That was my intent. Then I realized that the shark isn't featured on the mv web site. The astronaut is more or less **the image** for mv.

Reply:
If we want to use an astronaut, we should probably find a better astronaut. @elalish What should we use?

Reply:
I don't love that astronaut, but haven't yet found a replacement. I think any cool 3D model is good feature of model-viewer; the shark is totally fine.

Reply:
Shark is back.